test/org/opensolaris/opengrok/configuration/RuntimeEnvironmentTest.java

```diff
@@ -299,21 +299,19 @@ public void testIgnoredNames() {
     @Test
     public void testUserPage() {
         RuntimeEnvironment instance = RuntimeEnvironment.getInstance();
-        //String page = "http://www.myserver.org/viewProfile.jspa?username=";
-        String page = null;
-        assertNull(instance.getUserPage());
-        //instance.setUserPage(page.substring(5));
-        //assertEquals(page.substring(5), instance.getUserPage());
+        String page = "http://www.myserver.org/viewProfile.jspa?username=";
+        assertNull(instance.getUserPage());   // default value is null
+        instance.setUserPage(page.substring(5));
```

Comment:
I wonder what was the purpose of these `substring` operations. Perhaps they can be removed ?

Reply:
I can do that. They were just there from before.